### PR TITLE
Feature/bird details

### LINKED ISSUE #9: 3. USER STORY: As a user, when i click a bird card I am directed to a new page containing more information about the chosen bird.
- [ ] Bird cards are clickable
- [ ] Upon click event, user is routed to new page specific to that bird 
- [ ] Each bird has a specific URL
- [ ] Bird details page has scientific name
- [ ] Bird details page has common name
- [ ] Bird details page has a button saying "I SEE IT" with an icon of an eyeball
- [ ] "I SEE IT!" Button is clickable
- [ ] Nav Bar is visible with HOME button
- [ ] A "Home" Button is present that will take the user back to the home page

### PR DESCRIPTION
**Pull request by sydnerd**
# Pull Request Template
## Description

Create bird details page for bird selected.

## Type of change
- [ ] Bug fix 
- [X] New feature
- [ ] Refactoring

## Checklist:
- [X] My code follows the Turing's guidelines of this project
- [X] My changes generate no new errors/warnings

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -32,6 +33,16 @@ class App extends Component {
           <NavBar />
         </header>
         <Switch>
+          <Route path='/birds/:id' render={({ match }) => {
+            const bird = this.state.regionBirds.find(bird => bird.speciesCode === match.params.id)
+
+            if(!bird) {
+              return (<div>That bird doesn't exist!</div>)
```
Comment (LKessell) [inline src/components/App/App.js]: Would it make more sense for this to be a h2 or p, rather than a div?

File: src/components/Bird/Bird.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-    <div className="bird-card">
-      <h3>{commonName}</h3>
-      <p>{scientificName}</p>
-      <button>🔍</button>
-    </div>
+    <Link to={`/birds/${id}`}>
+      <div className="bird-card">
+        <h3>{commonName}</h3>
+        <p>{scientificName}</p>
+        <button>🔍</button>
```
Comment (LKessell) [inline src/components/Bird/Bird.js]: Are we planning to move this button to the bird details page?

File: src/components/App/App.js (hunk 1 above)
Comment (sydnerd) [reply src/components/App/App.js]: Sure! I'll change to an H2

**Review by k-atwhite — COMMENTED** [review]
Looks like that mag glass got moved. Good stuff

**Pull request merged**